### PR TITLE
Added Prom Native Clickhouse

### PR DESCRIPTION
**Pull request by hecomp**
- Added Prom Native Clickhouse

### CODE REVIEW

**Review by githubsands — APPROVED** [review]

**Review by codeboten — COMMENTED** [review]
please update the folder name to `prom-native-clickhouse` to remain consistent

**Review by codeboten — COMMENTED** [review]
Running `make run-client` gave me the following error: 

```
make run-client
docker run -it --rm --network clickhouse_integrations --link clickhouse-server:clickhouse-server yandex/clickhouse-client --host clickhouse-server
Unable to find image 'yandex/clickhouse-client:latest' locally
latest: Pulling from yandex/clickhouse-client
2f94e549220a: Pull complete
a72d8599d7c2: Pull complete
e9232762ed9d: Pull complete
29c8f4b1e77e: Pull complete
Digest: sha256:9ae2ee421c9c9f00406a39a1174276aa23abb7fceac13b40578b18eeaa9bc4d1
Status: Downloaded newer image for yandex/clickhouse-client:latest
docker: Error response from daemon: network clickhouse_integrations not found.
make: *** [run-client] Error 125
```

**Comment by hecomp** [comment]
> clickhouse_integrations

@codeboten make sure that yo run docker-compoes first?
Screen Shot 2022-07-26 at 12 41 07 PM

**Comment by codeboten** [comment]
> > clickhouse_integrations
> 
> @codeboten make sure that yo run docker-compoes first? Screen Shot 2022-07-26 at 12 41 07 PM

Yup, https://github.com/lightstep/opentelemetry-examples/pull/137/commits/7235074766a9c66c6481f30ba103d4d3fe49920f fixed the issue

File: collector/prom-native-clickhouse/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  ```
+  docker-compose up -d
+  ```
+* Run clickhouse client
+  ```
+  make run-client
+  ```
+  * Test DB
+  ```
+  CREATE DATABASE IF NOT EXISTS tutorial
```
Comment (codeboten) [inline collector/prom-native-clickhouse/README.md:29]: Copying and pasting this block as is causes the following error:

```
Syntax error: failed at position 40 ('CREATE') (line 2, col 1):

CREATE DATABASE IF NOT EXISTS tutorial
CREATE TABLE tutorial.xyz (a UInt8, d Date) ENGINE = MergeTree() ORDER BY (a) PARTITION BY toYYYYMM(d);
INSERT INTO tutorial.xyz values(8,'2022-07-25');
SELECT * FROM tutorial.xyz;
```

Might be worth a quick note about the "create database" statement needing to be done separately from the other commands.

**Comment by codeboten** [comment]
Seeing the following message in the collector logs:

```
otel-collector_1   | 2022-07-26T17:16:41.204Z	warn	internal/otlp_metricsbuilder.go:161	Failed to scrape Prometheus endpoint	{"kind": "receiver", "name": "prometheus", "scrape_timestamp": 1658855801201, "target_labels": "map[__name__:up instance:clickhouse-server:8123 job:clickhouse-server]"}
```

File: collector/prom-native-clickhouse/README.md (hunk 1 above)
Comment (hecomp) [reply collector/prom-native-clickhouse/README.md:29]: Good point, I'll modify to be as separate commands.

**Comment by hecomp** [comment]
> 

Hmm i'll look into it.

**Comment by hecomp** [comment]
> Seeing the following message in the collector logs:
> 
> ```
> otel-collector_1   | 2022-07-26T17:16:41.204Z	warn	internal/otlp_metricsbuilder.go:161	Failed to scrape Prometheus endpoint	{"kind": "receiver", "name": "prometheus", "scrape_timestamp": 1658855801201, "target_labels": "map[__name__:up instance:clickhouse-server:8123 job:clickhouse-server]"}
> ```

@codeboten issue addressed and put a fix.

File: collector/prom-native-clickhouse/Makefile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+DOCKER_NETWORK = prom-native-clickhouse_integrations
+
+run-client:
+	docker run -it --rm --network ${DOCKER_NETWORK} --link clickhouse-server:clickhouse-server yandex/clickhouse-client --host clickhouse-server
```
Comment (githubsands) [inline collector/prom-native-clickhouse/Makefile:4]: Remove this Makefile it is unnecessary

File: collector/prom-native-clickhouse/Makefile [hunk 3]
```diff
@@ -0,0 +1,4 @@
+DOCKER_NETWORK = prom-native-clickhouse_integrations
+
```
Comment (githubsands) [inline collector/prom-native-clickhouse/Makefile:2]: Remove this Makefile it is unnecessary

**Review by githubsands — CHANGES_REQUESTED** [review]
- Remove the Makefile 
-  Squash noise commits i.e "wips"

**Comment by githubsands** [comment]
i.e got this when running `run-client` ```
2f94e549220a: Pull complete 
a72d8599d7c2: Pull complete 
e9232762ed9d: Pull complete 
29c8f4b1e77e: Pull complete 
Digest: sha256:9ae2ee421c9c9f00406a39a1174276aa23abb7fceac13b40578b18eeaa9bc4d1
Status: Downloaded newer image for yandex/clickhouse-client:latest
docker: Error response from daemon: network prom-native-clickhouse_integrations not found.
make: *** [run-client] Error 125
ryan.vacek@MREM007E4155 prom-native-clickhouse % 
```

File: collector/prom-native-clickhouse/Makefile (hunk 3 above)
Comment (githubsands) [reply collector/prom-native-clickhouse/Makefile:2]: ran `make run-client` received error below.

**Pull request closed without merging**